README.md

```diff
@@ -1,2 +0,0 @@
-# User-Service
-Contains the combination of Patient service Caregiver service and Organization Service 
```

Comment:
We should add a basic README. What does the service do? How is it launched? What credentials need to be set? The other READMEs  of the organisation, patient-groups and patient-service should give a decent idea.